strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/RepositoryFiles.java

```diff
@@ -0,0 +1,72 @@
+package org.carlspring.strongbox.providers.io;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Set;
+import java.util.StringJoiner;
+import java.util.stream.Collectors;
+
+import org.carlspring.strongbox.artifact.coordinates.ArtifactCoordinates;
+
+public abstract class RepositoryFiles
```

Comment:
@fuss86 please note this utility class. Now you can check `isMetadata` or `isChecksum` for paths which is not present in File System.

Reply:
Cool! :)